src/vectorchord/advanced-features/prefetch.md

```diff
@@ -0,0 +1,31 @@
+# Prefetch <badge type="tip" text="since v0.4.0" />
+
+The default prefetch setting dramatically accelerates queries on large datasets by loading the necessary data from the disk into the memory cache before it is needed. It minimizes disk I/O latency during queries, allowing users to enjoy optimal performance with no configuration necessary.
```

Comment:
1. What's `dataset`? Is there such a thing in PostgreSQL?
2. What's `necessary data`? What data is not necessary?
3. What's `memory cache`? Is there such a thing in PostgreSQL?
4. What's `before it is needed`? Is it needed or not? If you mean using the data, then you simply can’t use it before reading.
5. Why it's `minimizes disk I/O latency`? We have reduced the latency, but it is far from minimized.